pkgs/by-name/pi/pista/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pista";
+  version = "378535f3e7e138110b6225616e0dc61066a5605a";
+
+  src = fetchFromGitHub {
+    owner = "oppiliappan";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-FlWDKz5B/sC+VCtJNmtCJTkxzeOJOMT9gZlG6UVGzKU=";
+    fetchSubmodules = true;
+  };
+
+  cargoHash = "sha256-/jjXwczzXHcK7gSkyjcxUo11S46yzDAWOHpxL7+mR1A=";
+
+  meta = with lib; {
+    description = "a simple {bash, zsh} prompt for programmers";
+    homepage = "https://github.com/oppiliappan/pista";
+    license = licenses.mit;
+    maintainers = with maintainers; [ benjajaja ];
+  };
```

```suggestion
  meta = {
    description = "Simple {bash, zsh} prompt for programmers";
    homepage = "https://github.com/oppiliappan/pista";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ benjajaja ];
  };
```

We are removing `with lib;` from `meta`.